### PR TITLE
GDD Update 1

### PR DESCRIPTION
**Pull request by NebulaeOfnir**
Added some basic mechanics info, especially details about resource management. Recommend doing Wiki before continuing so we can define terms as we use them in the document.

### CODE REVIEW

File: docs/GDD/OPENCOLD.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ Revision: 0.0.1
     - Total Costs with breakdown
 
 ## Influences (Brief)
-  - Influence #1
-    - Medium (Television, Games, Literature, Movies, etc.)
-    - /Explain why this is an influence in 1 Paragraph or less/
+  - Zero Sievert 
+    - Game
+    - Resource management system,
   - Influence #2
```
Comment (pigslyer) [inline docs/GDD/OPENCOLD.md:90]: Additional influence is definitely [Outer Wilds](https://store.steampowered.com/app/753640/Outer_Wilds/) given the "you have X time until the world dies and everything* restarts" USP.

File: docs/GDD/OPENCOLD.md [hunk 2]
```diff
@@ -119,17 +119,22 @@ Pretend that your were pitching your game to a executive going to the elevator.
...
+    - [Details] 
     - [How it works] /Describe in 2 Paragraphs or less/
-  - [Core Gameplay Mechanic #2]
-    - [Details] /Describe in 2 Paragraphs or less/
+  - Exploration
+    - [Details] 
     - [How it works] /Describe in 2 Paragraphs or less/
-  - [Core Gameplay Mechanic #3]
-    - [Details] /Describe in 2 Paragraphs or less/
+  - Resource Management
```
Comment (pigslyer) [inline docs/GDD/OPENCOLD.md:129]: 1. I feel a clearer divide between what an inventory resource and what a general resource is wouldn't be a bad idea. 
2. The order should also probably be shifted around (Health -> Heat -> Heat Producers -> Weapons and related -> Implants).
3. You repeated implants twice :P

**Review by pigslyer — CHANGES_REQUESTED** [review]
Other than that, looks good and is much appreciated. Will get the wiki up ASAP.

File: docs/GDD/OPENCOLD.md (hunk 2 above)
Comment (NebulaeOfnir) [reply docs/GDD/OPENCOLD.md:129]: 1. Done.
2. Done.
3. Oops. Done.

**Comment by NebulaeOfnir** [comment]
Fixed everything

**Comment by NebulaeOfnir** [comment]
Opa še energy dodala kr ni blo not. Gonna update wiki now

**Review by pigslyer — APPROVED** [review]

**Pull request merged**